tailwind.config.js

```diff
@@ -5,7 +5,7 @@ const plugin = require("tailwindcss/plugin");
 
 module.exports = {
     presets: [require("@frontify/arcade-tokens/tailwind")],
-    purge: ["./src/**/*.{js,ts,tsx}"],
```

Comment:
It looks like in `tailwind-config.ts` we are accessing this value. I am not sure if it is still handled correctly there or if it is even still needed.

Reply:
Good catch! I discussed it with Marco and we changed it in tailwind-config.ts as well 👍